pyaccel/new_lifetime.py

```diff
@@ -31,13 +35,13 @@ def lifetime(model, Vx, Vy, spread, P, I, n_bunches, emit, sig_S, acoplamento, a
     tau_am = pyaccel.optics.EquilibriumParameters.taue #radiationDamping;
     k =  pymodels.si.harmonic_number   #harmon;
     q = pyaccel.optics.EquilibriumParameters.overvoltage
-    qe = 1.60217662e-19; # electron charge [C]
-    r0 = 2.8179403227e-15; # classic electron radius [m]
-    c  = 2.99792458e8;  # speed of light [m/s]
-    Kb = 1.38e-23
+    qe = mathphys.constants.elementary_charge; # electron charge [C]
```

Comment:
@AnaLuisaAR, I think it is better to define all these physical constants as module variables instead of local variables. This way they don't need to be passed to the specific functions.